test/components/core_spec.js

```diff
@@ -95,22 +97,25 @@ describe('Core', function() {
           expect(fullScreenSpy).to.have.been.calledWith(this.core.activeContainer.el)
         })
       })
+
     })
 
     describe('when is in fullscreen', () => {
-      beforeEach(() => {
-        sinon.stub(Fullscreen, 'isFullscreen').value(() => true)
-      })
 
       it('calls Fullscreen.cancelFullscreen', () => {
         const spy = sinon.spy(Fullscreen, 'cancelFullscreen')
+        sinon.stub(Browser, 'isiOS').value(false)
+        sinon.stub(this.core, 'isFullscreen').returns(true)
+
         this.core.toggleFullscreen()
         expect(spy).to.have.been.called
+        
```

Comment:
There are a number of those empty lines at the end of blocks, please remove those.